### ISSUE TITLE
Uncaught TypeError: t.webVitals.getLCP is not a function

### ISSUE THREAD

**Issue by luzannew** [MindscapeHQ/raygun4js#393]
Hi,

Looks like there was a release tonight of RayGun and since that time we are now seeing this in our console. Yesterday we didn't have this issue.

Script is loaded from: https://cdn.raygun.io/raygun4js/raygun.min.js

```
raygun.min.js:formatted:16 Uncaught TypeError: t.webVitals.getLCP is not a function
    at e.attach (raygun.min.js:formatted:16)
    at Y.RealUserMonitoring.attach (raygun.min.js:formatted:2339)
    at e (raygun.min.js:formatted:1602)
    at ee (raygun.min.js:formatted:1605)
    at Y (raygun.min.js:formatted:1589)
    at raygun.min.js:formatted:1436
    at Object.init (raygun.min.js:formatted:1440)
    at b (raygun.min.js:formatted:2792)
    at raygun.min.js:formatted:2808
    at raygun.min.js:formatted:2809
```

When inspecting the console, we can indeed see that getLCP does not exists:

![image](https://user-images.githubusercontent.com/1443527/110301312-36075800-7ff8-11eb-8bc1-aa28bd4bbd9a.png)

Chrome Version 88.0.4324.192

We are using single-spa with import maps and it is required for us to import

`<script src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/6.1.1/extras/amd.min.js"></script>`

And due to this import, it will cause webVitals to not load correctly, because `typeof define === function` and `define.amd` is defined. See this file: https://github.com/MindscapeHQ/raygun4js/blob/master/src/raygun.rum/vendor/web-vitals.vendor.js

If we remove that file from the sandbox, it will load correctly and attach webVitals to the window element.

Reproducable scenario: https://codesandbox.io/s/raygun-singlespa-hw9wj?file=/src/index.js

**Comment by AdamLeBlanc** [comment]
Currently having the same issue across the board on all my projects. Causing major issues for customers. We had to disable it.

**Comment by SuzannahSmith** [comment]
Same problem here, when Raygun is enabled it is preventing entire pages from being loaded. Urgently need a fix!

**Comment by jessdeys** [comment]
Hi!! 

I'm Jess and I am the Delivery Manager here at Raygun. Thanks for getting in touch and letting us know about the issue you have been experiencing. 

We have just rolled back to a previous release and cleared any caching so upon refresh users should notice that the issue has been resolved. 

We are now going to work to resolve the issue in another release so I will keep this thread updated as we move forward with a new version.

**Comment by luzannew** [comment]
Hi Jess,

I can confirm the error is gone now. Thanks for reverting the release 👍 

Please let me know if I can test something for you in our setup.

**Comment by guillaumefielddata** [comment]
2.22.0 version of raygun4js npm package has been identified as containing breaking bugs and "rolled back". However I can confirm after some troubleshooting that 2.21.1 also contains this breaking bug which we could only resolve by force installing 2.21.0. This is quite problematic as any npm install operation happening with a package.json requiring `2.21.0` identified as `^2.21.0` will fetch the latest minor version being `2.21.1`. A version containing breaking changes.

**Comment by krishnaKapadia** [comment]
Hello, we have released version 2.22.1 which solves the issue above. Please upgrade the `raygun4js` npm package to the latest version. The CDN will automatically serve the latest version.

Cheers

**Comment by guillaumefielddata** [comment]
> Hello, we have released version 2.22.1 which solves the issue above. Please upgrade the `raygun4js` npm package to the latest version. The CDN will automatically serve the latest version.
> 
> Cheers

Thanks!
